src/components/border_editor/border_editor.xml

```diff
@@ -0,0 +1,70 @@
+<templates>
+  <t t-name="o-spreadsheet-BorderEditor" owl="1">
+    <div class="o-dropdown-content o-border-dropdown" t-on-click.stop="">
+      <div class="o-border-dropdown-section">
+        <div
+          t-foreach="BORDER_POSITIONS"
+          t-as="borderPositions"
+          t-key="borderPositions"
+          class="o-dropdown-line">
+          <span
+            t-foreach="borderPositions"
+            t-as="borderPosition"
+            t-key="borderPosition"
+            class="o-line-item"
+            t-att-class="{active:state.borderCommand == borderPosition[0]}"
+            t-att-title="borderPosition[0]"
+            t-att-foo="borderPosition[0]"
+            t-on-click.stop="() => this.setBorderPosition(borderPosition[0])">
+            <t t-call="{{borderPosition[1]}}"/>
+          </span>
+        </div>
+      </div>
+      <div class="o-divider"/>
+      <div class="o-border-dropdown-section">
+        <div
+          class="o-tool o-border-color-tool o-dropdown o-with-color"
+          title="Border color"
+          t-on-click.stop="(ev) => this.toggleDropdownTool('borderColorTool', ev)">
+          <span class="o-line-item" t-attf-style="border-color:{{state.borderColor}}">
+            <t t-call="o-spreadsheet-Icon.BORDER_COLOR"/>
+          </span>
+          <span class="o-line-item" t-attf-style="border-color:#fff0">
```

Comment:
![image](https://user-images.githubusercontent.com/91665701/202132316-020bb72c-d5c9-46de-97d6-8349a6ecbc1e.png)
Yes, this is used to avoid this kind of "color indicator". But it shoud maybe be put in a dedicated class